[DOC] Added core dev affiliations; minor updates to role holder page

fkiraly opened the pull request:
This PR makes the following changes to the "teams" page:

* changed header for better explanation of what the page is about - individuals and their roles, not the roles (role descriptions)
* added affiliations for all core developers: former affiliation of contribution and affiliation of primary job role (separate columns); for former core devs, affiliation of former contribution and affiliation of current primary job role
* added explanation of the two different affiliations to the table
* moved @jasonlines back to active
* moved @RNKuhns to inactive
* added @patrickzib 
* ordered lists by first name
* fixed incorrect GitHub ID of @MatthewMiddlehurst

fkiraly commented:
FYI @Lovkush-A who requested this for transparency.

fkiraly commented:
Tagging everyone for checking affiliations:
@ABostrom, @TonyBagnall, @chrisholder, @freddyaboulton, @goastler, @GuzalBulatova, @James-Large, @jasonlines, @mloning, @Lovkush-A, @aiwalter, @MatthewMiddlehurst, @RNKuhns, @SveaMeyer13, @Ayushmaanseth, @HYang1996, @matteogales, @big-o, @prockenschaub

big-o commented:
Hi, please can you just include my username only in the list. Also can you please remove my affiliation details; my contributions were only ever in a personal capacity and not affiliated with any company.

RNKuhns reviewed (approved):
Can you remove my affiliation too. I did it personally and there can be headaches related to me referencing where I work since it is government (and then I'd need a disclaimer for one). 

Also feel free to move me to inactive while your knocking this out.

MatthewMiddlehurst commented:
My username is incorrect.

RNKuhns reviewed (changes requested):
Meant my previous comment to request changes, not approve. I'm using the App so I must have clicked wrong. 

But the change is easy!

fkiraly commented:
ok, @RNKuhns, I separated the affil of the contribution from the primary affil to the best of my knowledge. Does this help with your headaches?

@MatthewMiddlehurst, I fixed your GitHub ID - it was like that before I reworked this.

fkiraly commented:
just noticed that @patrickzib was not on there. All new core devs should add themselves to this page!
Maybe worth a note in the core dev info pack.

fkiraly commented:
> Hi, please can you just include my username only in the list. Also can you please remove my affiliation details; my contributions were only ever in a personal capacity and not affiliated with any company.

I separated personal affiliation from contribution affiliation now.
I also removed your user name.

big-o commented:
> > Hi, please can you just include my username only in the list. Also can you please remove my affiliation details; my contributions were only ever in a personal capacity and not affiliated with any company.
> 
> @big-o: I separated personal affiliation from contribution affiliation now. I also removed your full name.

Please remove company details altogether for me - none of my contributions were ever company affiliated. It also causes issues if you link me with a company as I need to go through a process to make that kind of official affiliation.

fkiraly commented:
> Please remove company details altogether for me - none of my contributions were ever company affiliated. It also causes issues if you link me with a company as I need to go through a process to make that kind of official affiliation.

ok, done.

matteogales commented:
Can you please remove company details altogether for me? My contributions were in a personal capacity and not affiliated with any company, and I need to go through a process with my employer to make any public affiliation.

aiwalter commented on docs/source/about/team.rst:
```diff
@@ -50,34 +52,75 @@ Core Developers
+     - Freelancer; Shell Research (contracting)
    * - Martin Walter
      - :user:`aiwalter`
+     - Mercedes-Benz
+     - Mercedes-Benz
```
```suggestion
     - Individual
     - Individual
```

fkiraly replied:
Ok, I've changed this - is it ok (since former) or do I need to add more details?

Lovkush-A commented on docs/source/about/team.rst:
```diff
@@ -50,34 +52,75 @@ Core Developers
 
    * - Name
      - GitHub ID
+     - Current affiliation of contribution
+     - Affiliation of primary role
```
What does affiliation of primary role mean? Is 'role' referring to 'sktime governance roles' or to roles outside of sktime?

Lovkush-A replied:
Never mind, just saw the explanation at the bottom. Maybe move the explations of the columns above the table?

fkiraly replied:
I've added a better explanation, and the header is also clearer now, I hope?

Lovkush-A replied:
I don't think you have pushed the commits!

(Also, github has deleted a comment I made. In it, I suggested moving the explanation of the columns above the table rather than after the table.)

fkiraly replied:
??? I thought I did. You might be looking at an outdated version? Or I'm confused.

fkiraly replied:
>(Also, github has deleted a comment I made. In it, I suggested moving the explanation of the columns above the table rather than after the table.)

I don't think GitHub deletes comments. There might be something wrong with your browser cache or internet connectio?.

Lovkush-A reviewed (commented):
Do we want column for historical affiliations that helped with contributions to sktime?

fkiraly commented:
> Can you please remove company details altogether for me? My contributions were in a personal capacity and not affiliated with any company, and I need to go through a process with my employer to make any public affiliation.

Ok, I changed the rubric in the former core dev section to former affiliations.

TonyBagnall commented:
do we actually need  affiliations at all? people can look us up if they want, but it seems problematic to have a static view like this

RNKuhns commented:
Issue from my side is that I can only use affiliation with government agency in this type of context ( or giving a presentation) if it is part of "bio". I don't really know if this would constitute that. 

What about just having optional LinkedIn links for anyone who wants one?

big-o commented:
> do we actually need  affiliations at all? people can look us up if they want, but it seems problematic to have a static view like this

I agree, and furthermore I'd consider requesting for this PR to be removed from the project since it contains various people's personal information without necessarily having consent to publish it:

https://stackoverflow.com/a/18318431

Lovkush-A commented:
Probably makes sense to wipe out this PR given that it contains commits with personal info that people do not agree to being shared. Should move discussion about this into slack I guess.

fkiraly commented:
> I agree, and furthermore I'd consider requesting for this PR to be removed from the project since it contains various people's personal information without necessarily having consent to publish it:

[Important disclaimer: none of the below is legal advice, or a statement on behalf of, or endorsed by the Alan Turing Institute, or my employer. The below statements and opinions are entirely my own.]

Ok, @big-o, can you kindly be specific what information you want to have removed? All information about you, or any information that links you to your current employer?

Procedurally, if you want to go through full PR deletion without trace, not just closing: this is a function not supported by standard GitHub interfaces due to GitHub's transparency philosophy, so I think you might have to direct your request directly at the Alan Turing Institute who own and operate the GitHub organisation `alan-turing-institute`.

Another relevant factor is that `sktime` is - by its location and affiliation - an open research activity based in the academic space, and thus subject to freedom of research, and freedom of expression. Scientific best practice dictates that any research project must lay open any potential conflicts of interest, especially affiliation and resourcing.

Protection of personal information is wide-reaching, but as far as I understand it does not beat the academic requirement for laying bare potentially conflict of interest type connections in academic research - for instance, that Facebook can be, directly or indirectly, connected with the `sktime` project, through your person. You have been a core developer, i.e., a member of `sktime`'s governance, so in my opinion this is a potential conflict of interest that ought to be made transparent. Purging any mention and history of that seems quite an extreme step to me personally, especially since I've already redacted the current version.

I also see one thing arising from this discussion: we may want to be even clearer in our governance documents that transparency is an important cornerstone for an open source project like `sktime`, e.g., who is involved, who is directly/indirectly resourcing it, who has stakes in it. Personally, I think there might be justifiable exceptions, but the rule - as in all good open research - should be transparency.

fkiraly commented:
> Makes sense to wipe out this PR given that it contains commits with personal info that people do not agree to being shared. Should move discussion about this into slack I guess.

I'd agree that we need to discuss is with regards to individuals contributing in their personal capacity.
It is true that you can go on most GitHub IDs and get the information in the table easily by looking either there or on social media like LinkedIn, but I still should have asked before making the PR.
Well, to be fair, the topic came up on Slack (when you brought it up, @lovkush-a) but there reactions was not the same as above.

fkiraly commented:
> do we actually need affiliations at all? people can look us up if they want, but it seems problematic to have a static view like this

Hm, I was coming from the perspective of research transparency, and conflict of interest reporting requirements.
We can put it as a necessary requirement on existing core developers from a governance perspective.
Which might mean that a person without a good reason to hide their affiliations might not be able to become core developer.

fkiraly commented:
> Should move discussion about this into slack I guess.

Yes, let's have a priority discussion on Monday.

mloning reviewed (changes requested):
@fkiraly can you please remove my affiliation too. You publishing them here is a bad idea, especially when people have made contributions in other capacities. I think we should close this PR. Generally, people should be able to opt in, rather than one of us publishing this here.

mloning commented:
Closed until further discussion.
